guidelines.md

```diff
@@ -0,0 +1,91 @@
+## GIT AND GITHUB
+***
+Before continuing we want to clarify the difference between Git and Github. Git is a version control system(VCS) which is a tool to manage the history of our Source Code. GitHub is a hosting service for Git projects.
+
+We assume you have created an account on Github and installed Git on your System.
+
+Now tell Git your name and E-mail (used on Github) address.
+
+``` $ git config --global user.name "YOUR NAME" ```
+```$ git config --global user.email "YOUR EMAIL ADDRESS"```
+This is an important step to mark your commits to your name and email.
+
+### FORK A PROJECT -
+***
+You can use github explore - https://github.com/explore to find a project that interests you and match your skills. Once you find your cool project to workon, you can make a copy of project to your account. This process is called forking a project to your Github account. On Upper right side of project page on Github, you can see -
+
+<p align="center">  <img  src="https://i.imgur.com/P0n6f97.png">  </p>
+
+Click on fork to create a copy of project to your account. This creates a separate copy for you to workon.
+
+### FINDING A FEATURE OR BUG TO WORKON - 
+***
+Open Source projects always have something to workon and improves with each new release. You can see the issues section to find something you can solve or report a bug. The project managers always welcome new contributors and can guide you to solve the problem. You can find issues in the right section of project page.
+
+<p align="center">  <img  src="https://i.imgur.com/czVjpS7.png">  </p>
+
+### CLONE THE FORKED PROJECT -
+***
+You have forked the project you want to contribute to your github account. To get this project on your development machine we use clone command of git.
+
+```$ git clone https://github.com/<your-account-username>/<your-forked-project>.git```
+Now you have the project on your local machine.
+
+### ADD A REMOTE (UPSTREAM) TO ORIGINAL PROJECT REPOSITORY 
+***
+Remote means the remote location of project on Github. By cloning, we have a remote called origin which points to your forked repository. Now we will add a remote to the original repository from where we had forked.
+
+```$ cd <your-forked-project-folder>```
+```$ git remote add upstream https://github.com/<author-account-username>/<project>.git```
+You will see the benefits of adding remote later.
+
+### SYNCHRONIZING YOUR FORK -
+***
+Open Source projects have a number of contributors who can push code anytime. So it is necessary to make your forked copy equal with the original repository. The remote added above called Upstream helps in this.
+
+```$ git checkout master```
+```$ git fetch upstream```
+```$ git merge upstream/master```
+```$ git push origin master```
```

Comment:
write all the git commands under the same triple quotes sign as shown below.
 - start quotes
 - git command one
 - git command two
 - git command three
 - git command four
 - end quotes